build.sbt

```
@@ -1,19 +1,14 @@
-import sbtcrossproject.CrossPlugin.autoImport.crossProject
-import sbtcrossproject.CrossPlugin.autoImport.CrossType
 import com.typesafe.tools.mima.core._
 
-ThisBuild / baseVersion := "3.1"
+ThisBuild / tlBaseVersion := "3.1"
 
 ThisBuild / organization := "com.comcast"
 ThisBuild / organizationName := "Comcast Cable Communications Management, LLC"
 
-ThisBuild / homepage := Some(url("https://github.com/comcast/ip4s"))
 ThisBuild / startYear := Some(2018)
 
-ThisBuild / publishGithubUser := "mpilquist"
-ThisBuild / publishFullName := "Michael Pilquist"
-
 ThisBuild / developers ++= List(
+  Developer("mpilquist", "Michael Pilquist", "@mpilquist", url("https://github.com/mpilquist")),
```

These can be shortened.
```suggestion
  tlGitHubDev("mpilquist", "Michael Pilquist"),
```